app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -1,38 +1,36 @@
 package edu.uc.hickmadc.customnotifications
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
 import androidx.navigation.Navigation
 import androidx.navigation.findNavController
 import androidx.navigation.ui.NavigationUI
 import com.google.android.material.bottomnavigation.BottomNavigationView
 import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 
-//import edu.uc.hickmadc.customnotifications.ui.main.MainFragment
-
 class MainActivity : AppCompatActivity() {
 
-    //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
+    /*
+        Temporary data
+     */
+    val language = arrayOf<String>("Brush Teeth", "Eat Breakfast", "Do schoolwork")
     val description = arrayOf<String>(
         "Brush teeth for 2 minutes",
         "Eat so you don't feel like shit",
-        "")
+        "So you can pass code reviews!"
+    )
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
 
-        val lView= findViewById<ListView>(R.id.listView)
-        val myListAdapter = MyListAdapter(this,language,description)
+        val lView = findViewById<ListView>(R.id.listView)
+        val myListAdapter = MyListAdapter(this, language, description)
         lView.adapter = myListAdapter
         if (savedInstanceState == null) {
             setupBottomNavigationBar()
-//            supportFragmentManager.beginTransaction()
-//                .replace(R.id.container, MainFragment.newInstance())
-//                .commitNow()
```

Comment:
Just remove it when you commit it, there's no need to comment out whole methods.

Reply:
Yes